test/testcafe/screenshot-test.js

```diff
@@ -0,0 +1,11 @@
+import { expect } from 'chai';
+import { statSync } from 'fs';
+
+fixture `Screenshot`
+    .page('https://google.com');
+
+test('Take screenshot', async t => {
+    var screenshotPath = await t.takeScreenshot();
```

Comment:
Yes, it returns path to the captured screenshot: https://github.com/DevExpress/testcafe/blob/master/src/screenshots/capturer.js#L95. It's not documented both in the old and the new docs, but AFAIR `act.screenshot` worked in the same way too.